api.md

```diff
@@ -214,6 +217,19 @@ millisecond, nanosecond, etc.).  So if your time column is the number
 of milliseconds since the UNIX epoch, and you wish to each chunk to
 cover 1 day, you should specify `chunk_time_interval => 86400000`.
 
+The units of `chunk_target_size` follow the format of a ["Numeric with
+Unit"][memory-units] memory settings parameter in PostgreSQL. Note
+that this size includes indexes when using the default adaptive
+chunking algorithm.
```

Comment:
"Be careful with specifying a plain number ("100000000") rather than one with units ("100MB"), as such plain numbers can correspond to an unexpected unit, e.g., the number of memory pages (in which case 100,000,000 * 8K pages = 800GB)."